packages/shylu/shylu_dd/frosch/src/SchwarzOperators/FROSch_IPOUHarmonicCoarseOperator_def.hpp

```diff
@@ -170,6 +174,8 @@ namespace FROSch {
         FROSCH_TIMER_START_LEVELID(resetCoarseSpaceBlockTime,"IPOUHarmonicCoarseOperator::resetCoarseSpaceBlock");
         FROSCH_ASSERT(dofsMaps.size()==dofsPerNode,"dofsMaps.size()!=dofsPerNode");
         FROSCH_ASSERT(blockId<this->NumberOfBlocks_,"Block does not exist yet and can therefore not be reset.");
+        this->dofs = nullSpaceBasis->getNumVectors();
```

Comment:
If this is the dimension of the null space, the name `dofs` may be misleading?

Reply:
`this->dofs` are the degrees of freedom on the coarse level; I will change the name of this variable.